autoPyTorch/api/base_task.py

```diff
@@ -102,13 +102,16 @@ def send_warnings_to_log(
 class BaseTask:
     """
     Base class for the tasks that serve as API to the pipelines.
+
     Args:
-        seed (int), (default=1): seed to be used for reproducibility.
-        n_jobs (int), (default=1): number of consecutive processes to spawn.
+        seed (int), (default=1):
```

Comment:
This looks better? (I could not find an example of default in google doc-string)
```suggestion
        seed (int: default=1):
```